portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintruleset.js

```diff
@@ -472,11 +472,17 @@ var LinearInterpRuleSet = (function () {
 	this.value_key = params.value_key;
 	this.value_range = params.value_range;
 	this.log_scale = params.log_scale; // boolean
-	this.inferred_value_range;
+
+	this.rangeTypes = {
+		'SIGNED': 'SIGNED',
+		'UNSIGNED': 'UNSIGNED',
+		'NON_POSITIVE': 'NON_POSITIVE'
```

Comment:
@rnugraha ok, the three new names you suggest are better (plus: a small documentation line about the "inclusive 0" part - this would make it clear enough). Thanks.